fvm/bootstrap.go

```diff
@@ -476,6 +473,14 @@ func (b *bootstrapExecutor) Execute() error {
 		return err
 	}
 
+	// deploy flow transaction scheduler contract to the service account
+	b.deployFlowTransactionScheduler(service, &env)
+
+	err = expectAccounts(systemcontracts.ScheduledTransactionExecutorAccountIndex)
```

Comment:
Increasing the amount of addresses created during bootstrapping usually has consequences for the emulator and anyone using the emulator. We need to make sure we announce this properly.